ural/should_be_rss_url.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Ural Should Be RSS URL
+# =============================================================================
+#
+# Function returning whether the given url should be a rss feed url.
+#
+import re
+
+from ural.utils import urlpathsplit
+from ural.utils import urlsplit
+import ural.get_hostname
+
+PATTERN_QUERY = re.compile(r"(?:format|type)\=(?:xml|feed|atom|rss)", re.I)
+PATTERN_FILE_OBVIOUS = re.compile(
+    r"(?:news|feed|rss|atom|index|blog)s?\.(?:xml|atom|rss|php|json)$", re.I
+)
+PATTERN_FILE = re.compile(r".+\.(?:rss|atom)")
+PATTERN_KEYWORD = re.compile(
+    r"(?:\/|\.|-|_)(?:rss|feed|atom)s?[0-9]*(((?:s|\/|\.|-|_))||^)", re.I
+)
+
+
+def should_be_rss_url(url):
```

Comment:
Ural has a function named `could_be_html` so I guess this one should be named `could_be_rss`.